crates/holochain_types/src/dht_op.rs

```diff
@@ -111,6 +113,24 @@ pub enum DhtOp {
     RegisterRemoveLink(Signature, header::DeleteLink),
 }
 
+impl kitsune_p2p_dht::prelude::OpRegion for DhtOp {
+    fn loc(&self) -> Loc {
+        self.dht_basis().get_loc()
+    }
+
+    fn timestamp(&self) -> Timestamp {
+        self.timestamp()
+    }
+
+    fn region_data(&self) -> RegionData {
+        unimplemented!()
+    }
+
+    fn bound(_timestamp: Timestamp, _loc: kitsune_p2p_dht::Loc) -> Self {
+        unimplemented!()
+    }
```

Comment:
are these `unimplemented!`s ok?

Reply:
Yeah, I don't think this trait is even used anywhere. It was an attempt to generalize the op type from kitsune's perspective, but I didn't wind up actually fully going forward with the idea...could be good to either actually fully do this, or clean this up later. (Seems hard to imagine having the time to refactor kitsune any time soon)